sm-core-model/src/main/java/com/salesmanager/core/model/catalog/category/image/CategoryImage.java

```diff
@@ -0,0 +1,84 @@
+package com.salesmanager.core.model.catalog.category.image;
+
+import com.salesmanager.core.constants.SchemaConstant;
+import com.salesmanager.core.model.catalog.category.Category;
+import com.salesmanager.core.model.generic.SalesManagerEntity;
+
+import javax.persistence.*;
+import java.io.InputStream;
+import java.util.UUID;
+
+/**
+ * @author Mostafa Saied <mostafa.saied.fci@gmail.com>
+ */
+
+@Entity
+@Table(name = "CATEGORY_IMAGE", schema= SchemaConstant.SALESMANAGER_SCHEMA)
```

Comment:
why do you need to define schema here?

Reply:
That's how all the entities in the project are created, just sticked to the same style.